test/interface.spec.js

```diff
@@ -102,40 +102,30 @@ describe('interface-ipfs-core tests', () => {
 
   tests.filesRegular(defaultCommonFactory, {
     skip: [
-      // .add
-      isNode ? null : {
-        name: 'should add a nested directory as array of tupples',
-        reason: 'FIXME https://github.com/ipfs/js-ipfs-http-client/issues/339'
-      },
-      isNode ? null : {
-        name: 'should add a nested directory as array of tupples with progress',
-        reason: 'FIXME https://github.com/ipfs/js-ipfs-http-client/issues/339'
-      },
-      // .addPullStream
-      isNode ? null : {
-        name: 'should add pull stream of valid files and dirs',
-        reason: 'FIXME https://github.com/ipfs/js-ipfs-http-client/issues/339'
-      },
-      // .addReadableStream
-      isNode ? null : {
-        name: 'should add readable stream of valid files and dirs',
-        reason: 'FIXME https://github.com/ipfs/js-ipfs-http-client/issues/339'
-      },
+      // // .add
+      // isNode ? null : {
+      //   name: 'should add a nested directory as array of tupples',
+      //   reason: 'FIXME https://github.com/ipfs/js-ipfs-http-client/issues/339'
+      // },
+      // isNode ? null : {
+      //   name: 'should add a nested directory as array of tupples with progress',
+      //   reason: 'FIXME https://github.com/ipfs/js-ipfs-http-client/issues/339'
+      // },
+      // // .addPullStream
+      // isNode ? null : {
+      //   name: 'should add pull stream of valid files and dirs',
+      //   reason: 'FIXME https://github.com/ipfs/js-ipfs-http-client/issues/339'
+      // },
+      // // .addReadableStream
+      // isNode ? null : {
+      //   name: 'should add readable stream of valid files and dirs',
+      //   reason: 'FIXME https://github.com/ipfs/js-ipfs-http-client/issues/339'
+      // },
```

Comment:
Please delete commented code.

Reply:
it was on purpose just want to make sure these pass in the CI, they were skipped for a reason. But they seem to pass now. 
ill remove the comments after CI green.